OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -22,6 +22,14 @@ class GivesExperienceInfo : ITraitInfo
 		[Desc("Grant experience for team-kills.")]
 		public readonly bool FriendlyFire = false;
 
+		[Desc("Modifier (in percent) with which the `Experience` value will be multiplied",
+			"before being granted to an actor.")]
+		public readonly int ActorExperienceModifier = 10000;
```

Comment:
It *is* a percentage. Actors really do get 100x the value of the unit they killed counted towards their experience. If we didn't inflate the value, you couldn't increase experience by less than the killed actor's value (ie. anything below 100%).